CMakeLists.txt

```diff
@@ -63,6 +63,14 @@ if(coverage)
   set(CMAKE_SHARED_LIBRARY_CREATE_CXX_FLAGS "${CMAKE_SHARED_LIBRARY_CREATE_CXX_FLAGS} ${GCC_COVERAGE_COMPILE_FLAGS}")
 endif()
 
+#--- Enable build timing -----------------------------------------------------------------------
+if (build_timing)
+  # FIXME: This currently will override the use of ccache if -Dbuild_timing=On -Dccache=On is passed.
```

Comment:
It probably is, but I do not see a lot of benefit in combining the timing info with `ccache`. In ccache mode we want to be as fast as possible, adding one `time` command to every ccache invocation seems like an overkill.